overlord/snapstate/snapstate.go

```diff
@@ -314,6 +314,29 @@ func refreshRetain(st *state.State) int {
 	return retain
 }
 
+func MaxInhibitionTime(st *state.State) time.Duration {
+	// deduct 1s so it doesn't look confusing initially when two notifications
```

Comment:
My personal feeling is that this -1s part is somewhat misplaced. I don't grok the piece that is responsible for the display or why it would display twice but it feels like a layering problem.

At the same time, not something I feel strongly about if the fix is very hard.

Reply:
I just adjusted the original code. That was exactly the same thing done before when creating the constant.